quipucords/fingerprinter/runner.py

```diff
@@ -508,8 +508,8 @@ def _process_source(self, source):
                 fingerprint = self.process_facts_for_datasource(
                     source_type, source, fact
                 )
-            except KeyError:
-                self.scan_task.log_message.error(
+            except NotImplementedError:
```

Comment:
Where did the `KeyError` transform into `NotImplementedError`? Or was this already wrong before? Did this result from changing `serializer.is_valid(raise_exception)`?

Reply:
It was already wrong before. I should have saved this change in a separate commit.

here's the reasoning
- [this place](https://github.com/quipucords/quipucords/blob/10c64d08060ebace0a483b2b1b22980248f1b1e4/quipucords/fingerprinter/runner.py#L482-L487) is where the exception is thrown.
- [15 months ago](https://github.com/quipucords/quipucords/commit/dd705ba265fe0c93457c85541a7d4fc6a1968102) this was raising a `KeyError` instead of `NotImplementedError`.